tests/unit_tests/percolator/CMakeLists.txt

```diff
@@ -2,7 +2,7 @@
 # If GoogleTest is not installed, build and install a private copy.
 # Skip this step if GOOGLE_TEST is not defined.
 if(GOOGLE_TEST)
-  find_package(GoogleTest REQUIRED)
+  find_package(GTest)# REQUIRED)
```

Comment:
We had an old helper script called FindGoogleTest.cmake that retrieved the right install location of GoogleTest. Since that CMake made one of their own, which comes with cmake, called FindGTest. It is more sustainable to use their version of the script.

It has to be optional. If the libraries are uninstalled, we want the script GTestExtProject to kick-in, rather than the build halting.